### DISCUSSION TITLE
Add top 10 netflix

### DISCUSSION THREAD

**Discussion by yalagin** [seerr-team/seerr#200] [Ideas]
Add a convenient way to watch top 10 form Netflix 

https://top10.netflix.com/austria/tv

**Comment by DoomzKid** [comment]
Would be cool the have something like that indeed, That you have a quick overview about the top 10 shows/movies from popular service and you could use "Follow" button so it will keep the top 10 in sync for the service you want.

![signal-2025-05-04-131734](https://github.com/user-attachments/assets/9ac4e3aa-6364-4e9a-a970-45a2e30c4a08)
